tide_core.module

```diff
@@ -189,3 +190,19 @@ function tide_core_entity_bundle_create($entity_type_id, $bundle) {
     }
   }
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function tide_core_preprocess_status_messages(&$variables) {
+  if (isset($variables['message_list']['error']) && !empty($variables['message_list']['error'])) {
+    foreach ($variables['message_list']['error'] as &$error_message) {
+      if ($error_message instanceof Markup) {
+        $error_message = (string) $error_message->__toString();
+        if (strpos($error_message, 'Validation error on collapsed paragraph field_paragraph_link') !== FALSE) {
+          $error_message = t('A link in <i>Related links</i> field on this page is broken. Please update or remove the link and the page will save.');
```

Comment:
```suggestion
          $error_message = t('A link in <i>Related links</i> field on this page is broken. Please update or remove the link and retry.');
```

I'm suggesting this change because we can't guarantee that the node will save once the user has made a change to the Related Links field.

Reply:
changed and committed.